spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -441,4 +441,95 @@
         .to raise_error("#{ErrorMessage::INVALID_TESTERS} \nEmails: #{emails} \nGroups: #{group_ids}")
     end
   end
+
+  describe '#add_testers' do
+    let(:headers) { { 'Authorization' => 'Bearer auth_token', 'Content-Type' => 'application/json' } }
+
+    it 'is successful' do
+      emails = %w[1@foo.com 2@foo.com]
```

Comment:
TIL `%w` is a thing 🤯

Reply:
Ha yeah rubocop was adamant that I use it 😆

Reply:
Oh good to know!